accessibility/index.html

```diff
@@ -0,0 +1,550 @@
+                        
+                <!-- HOW TO ACTIVATE VOICEOVER FOR OSX START -->
+                        
+                <section>
+						<h2>Activate <span class="gdi">VoiceOver</span><br/>Screen Reader - OSX</h2>
+                    <small><span class="gdi">Instructions</span> for activating VoiceOver for OSX.</small>
+                    <p><strong><span class="gdi">Activate</span> VoiceOver</strong> from <span class="gdi">System Preferences</span> > <span class="gdi">Accessibility</span> > <span class="gdi">Enable VoiceOver</span> checkbox</p>
+				</section>
+                <aside class="notes">
+				    Instructions for using VoiceOver on an OSX device once it has been enabled. - Demo and Group Activity
+                </aside>
+                        
+                <!-- HOW TO ACTIVATE VOICEOVER FOR OSX END -->
+                        
+				<!-- IMPORTANCE OF SEMANTIC HTML START -->
+				
+				<section>
+				    <h2>Importance of</br><span class="gdi">Semantic HTML Elements</span></h2>
+                    <small>The accessibility API is part of a device's operating system and represents objects in a user interface, exposing information about each object within the application or Webpage</small>
+
+				    <p>Semantic HTML elements are immediately identified by the accessibility API and interpreted by screen readers. Non-semantic elements convey no information to the accessibility API.</p>
+                    <small>OSX Accessibility API Demo</small>
+                
+				</section>
+				
+				<!-- IMPORTANCE OF SEMANTIC HTML END -->
+                        
+                <!-- EXAMPLES OF SEMATIC VS NON-SEMANTIC HTML START -->  
+                    
+				<section>
+                    <h2><span class="gdi">Semantic</span> Elements</h2>
+                    <pre><code data-trim contenteditable> 
+                    
+<!-- Link Example -->
+ <a href="http://www.someurl.com">Go to some Web page</a>
+
+ <!-- Button Example -->
+ <button>Click me</button>
+ 
+                    </code></pre>
+                    <br/>
+                    <h2><span class="gdi">Non-Semantic</span> Elements</h2>
+                    <small>Divs styled with CSS to look semantic</small>
+                    <pre><code data-trim contenteditable>    
+                    
+ <!-- Link Example -->
+ <div class="link-class">Go to some Web page</div>
+
+ <!-- Button Example -->
+ <div class="button-class">Click Me</div> 
+ 
+                    </code></pre>
+                    
+                </section>
+  
+                <!-- EXAMPLES OF SEMATIC VS NON-SEMANTIC HTML END -->  
+				
+				<!-- Exercise 01 START -->
+
+				<section>
+					<h3>Exercise One</h3>
+          			<p>Use VoiceOver to read content that uses <br/><strong>Non-Semantic</strong> HTML elements</p>
+		  			<small><a href="https://inclusiveThinking.jsbin.com/regipe/edit?html,css,output" target="_blank">Open Exercise</a></small>
```

Comment:
Sorry, I had made all bins private when I was building this out. I will
make everything public

On Sun, Oct 4, 2015 at 12:02 PM, Brenda Jin <notifications@github.com>
wrote:

> In accessibility/index.html
> <https://github.com/pamelafox/teaching-materials/pull/169#discussion_r41101041>
> :
>
> > +
> > + <!-- Button Example -->
> > + <div class="button-class">Click Me</div>
> > +
> > +                    </code></pre>
> > +
> > +                </section>
> > +
> > +                <!-- EXAMPLES OF SEMATIC VS NON-SEMANTIC HTML END -->
> > +				
> > +				<!-- Exercise 01 START -->
> > +
> > +				<section>
> > +					<h3>Exercise One</h3>
> > +          			<p>Use VoiceOver to read content that uses <br/><strong>Non-Semantic</strong> HTML elements</p>
> > +		  			<small><a href="https://inclusiveThinking.jsbin.com/regipe/edit?html,css,output" target="_blank">Open Exercise</a></small>
>
> Can you test this link in an incognito window? I wasn't able to open it
> correctly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pamelafox/teaching-materials/pull/169/files#r41101041>
> .
>